core/src/main/java/org/eclipse/hono/util/TenantConstants.java

```diff
@@ -89,6 +89,16 @@
      */
     public static final String MAX_CONNECTIONS = "max-connections";
 
+    /**
+     * The default value for the maximum data volume to be allowed for a tenant per month, which implies no limit.
+     */
+    public static final long DEFAULT_MAX_BYTES_PER_MONTH = -1;
+
+    /**
+     * The name of the property that contains the maximum data volume to be allowed for a tenant per month.
+     */
+    public static final String MAX_BYTES_PER_MONTH = "max-bytes-per-month";
```

Comment:
This should better be renamed to `FIELD_MAX_BYTES_PER_MONTH` to make clear that this constant defienes the name of the field only and not the value
This also holds true for the *MAX_CONNECTIONS* constant ...

Reply:
@sophokles73 Done :-)